scss/_patterns_navigation-reduced.scss

```diff
@@ -0,0 +1,120 @@
+@mixin vf-p-navigation-reduced {
+  @media (min-width: $breakpoint-navigation-threshold - 1) {
+    .p-navigation--sliding.is-reduced.has-search-open {
+      .p-navigation__nav .p-navigation__items {
+        display: inline-flex;
+      }
+
+      .p-navigation__search {
+        align-items: center;
+        background-color: #262626;
+        display: flex;
+        justify-content: center;
+        left: 0;
+        padding: 0.625rem 1rem;
+        position: absolute;
+        top: 2rem;
+        width: 100%;
+
+        .p-search-box {
+          margin: 0;
+          max-width: 50rem;
+        }
+      }
+
+      .p-navigation.is-secondary {
+        display: none;
+      }
+    }
+  }
+
+  .p-navigation--sliding.is-reduced {
+    position: relative;
+
+    .p-navigation__items:first-child {
+      margin-left: -1rem;
+    }
+
+    .p-navigation__logo-tag {
+      display: none;
+    }
+
+    .p-navigation__tagged-logo {
```

Comment:
Kind of related question to the one above… We never show a tagged logo in reduced nav, right? So it's a bit confusing, that we have markup for it, when it could simply be treated as a link or standard navigation item?